### PR TITLE
Route Component

### PR DESCRIPTION
**Pull request by mrym-rhm**
Tapsi Route component created based on https://github.com/Tap30/web-components/issues/73 issue.
close: https://github.com/Tap30/web-components/issues/73

image

Also "disabled" status added to Row component and used in Route component:

image

image

### CODE REVIEW

File: src/row/row.ts [hunk 1]
```diff
@@ -55,27 +57,38 @@ export class Row extends LitElement {
...
       </svg>
     `
   }
 
+  private renderOverlay() {
+    return this.disabled
+      ? html`
+        <div class="overlay"/>
+      `
+      : '';
```
Comment (amir78729) [inline src/row/row.ts:72]: I think you can use `nothing`

File: src/row/row.ts [hunk 2]
```diff
@@ -2,9 +2,11 @@ import {html, LitElement, PropertyValues} from "lit";
 import {property} from "lit/decorators.js";
 
 export class Row extends LitElement {
-  @property({ reflect: true }) size: "standard" | "compact" = "standard";
```
Comment (amir78729) [inline src/row/row.ts:5]: Please format this file based on ESLint

File: src/row/row.style.ts [hunk 3]
```diff
@@ -72,4 +73,14 @@ export default css`
...
+
+  .overlay {
+    background-color: #FFF;
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    opacity: 40%;
+  }
+
+  :host([act])
```
Comment (amir78729) [inline src/row/row.style.ts:85]: What does this line do?

**Comment by mrym-rhm** [comment]
closes https://github.com/Tap30/web-components/issues/73

**Comment by amir78729** [comment]
@mrym-rhm Please format your code using elsint

**Comment by amir78729** [comment]
@mrym-rhm Please add JSDoc if it's possible for you

**Pull request merged**